app/views/static_pages/index.html

```diff
@@ -2,21 +2,47 @@
 <!-- <img src="Hong Kong 01 06 Central District From Victoria Peak Nighttime.jpg" alt="Central" class="fp-image"> -->
 
 <div id="tab-locations">
-  this will be the locations tab.
-
-  <div class="locations-img-central" >
-    <button class="btn btn-default btn-lg" type="button">Central
+  <div class="option-page-text">
+  Please pick a location!
+  </div>
+  <div class="locations-img-central col-xs-3">
```

Comment:
* Boostrap grid has 12 columns. So inside each row the total of the col-xx-x should be 12, for each screen size: col-xs-3 + col-xs-4 + col-xs-3 = 10 !
* Also note that this should be true for each of Bootstrap's screen sizes: lg, md, sm, xs